src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs

```diff
@@ -483,10 +483,10 @@ public IClassificationLoss CreateComponent(IHostEnvironment env)
         }
 
         /// <summary>
-        /// Predict a target using a linear regression model trained with the <see cref="OnlineGradientDescentTrainer"/> trainer.
+        /// Predict a target using a linear regression model trained with the <see cref="OnlineGradientDescentTrainer"/> and advanced options.
```

Comment:
>and [](start = 121, length = 3)

specifying

Reply:
i think "and" is good enough and reads well. This is the language pattern I've used for all trainers so far, so I don't want to make recursive changes to completed trainers unless there's an error or major improvement.

---
In reply to: [268037979](https://github.com/dotnet/machinelearning/pull/3059#discussion_r268037979) [](ancestors = 268037979)